packages/terra-dropdown-button/tests/jest/DropdownList.test.jsx

```diff
@@ -89,4 +94,54 @@ describe('Dropdown List', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should set the aria-label property to empty on keydown', () => {
+    const listRefMock = {
+      childNodes: [
+        { focus: jest.fn() },
+        { focus: jest.fn() },
+        { focus: jest.fn() },
+      ],
+    };
+
+    const eventMock = {
+      key: 'ArrowDown',
+      keyCode: 40,
+      target: { textContent: 0 },
+      preventDefault: jest.fn(),
+      stopPropagation: jest.fn(),
+    };
+
+    const wrapper = shallowWithIntl(
+      <IntlProvider locale="en" messages={translationsFile}>
+        <DropdownList id="dropdownList" requestClose={() => { }}>
+          <Item id="firstItem" label="1st Option" onSelect={() => { }} />
+          <Item label="2nd Option" onSelect={() => { }} />
+          <Item label="3rd Option" onSelect={() => { }} />
+        </DropdownList>
+      </IntlProvider>,
+    ).dive().dive();
+
+    wrapper.instance().listRef = listRefMock;
+    const firstListItem = wrapper.find('#firstItem');
+    const firstListItemAriaLabelValue = firstListItem.props()['aria-label'];
+    const expectedAriaLabelValueInitial = `${translationsFile['Terra.dropdownButton.expanded']}1st Option,(1 of 3)`;
+    const expectedAriaLabelValueInitialEDGE = `${translationsFile['Terra.dropdownButton.expanded']}1st Option`;
+    if (SharedUtil.isMac()) {
+      expect(firstListItemAriaLabelValue).toEqual(expectedAriaLabelValueInitial);
+    } else {
+      expect(firstListItemAriaLabelValue).toEqual(expectedAriaLabelValueInitialEDGE);
+    }
```

Comment:
Unit tests should NOT have logic around expect statements. This should be 2 separate unit tests and you can mock the return value for `SharedUtil.isMac()`. 

https://jestjs.io/docs/mock-functions
https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options

Reply:
Updated.Thanks